package.json

```diff
@@ -106,7 +104,8 @@
     "stylelint": "^13.13.1",
     "stylelint-config-standard": "^20.0.0",
     "stylelint-scss": "^3.16.0",
-    "yargs": "^15.3.1"
+    "yargs": "^15.3.1",
+    "nodelist-foreach-polyfill": "^1.2.0"
```

Comment:
small one, but we can actually remove the polyfill now because [it was only IE that didn't have `NodeList.forEach`](https://caniuse.com/mdn-api_nodelist_foreach) 

just a random drive by skim since I left some tabs open - will review properly in morning!

Reply:
Ah, I followed the TODO in the [common.js file](https://github.com/hmrc/hmrc-frontend/blob/main/src/common.js), is this not a valid approach?